### ISSUE TITLE
Add option to disable email notifications

### ISSUE THREAD

**Issue by MarinescuLucia** [Codeinwp/wp-maintenance-mode#338]
A user asked if it's possible to disable the email notifications that are sent to the admin when a person subscribes to the form from the maintenance/coming soon page. Maybe we could add an option to be able to do this.

As an alternative solution, I think we could create a doc explaining how to achieve this with custom code.

Suggested [here](https://wordpress.org/support/topic/disable-email-notifications-12/).

**Comment by girishpanchal30** [comment]
Hello @MarinescuLucia,

I can't find a filter to skip the admin notification, it appears that the otter block plugin handles admin notifications.

@arinaturcu @HardeepAsrani Do you have any idea to skip the admin notification?

Thanks

**Comment by HardeepAsrani** [comment]
@girishpanchal30 No, we always send email as we don't store them anywhere, just send them straight away to the user. It makes sense to have them if the user is using the form only for a subscription. Will move this ticket to Otter repository.

Thanks @MarinescuLucia for reporting this.
